utils/build/docker/golang/app/chi.go

```diff
@@ -128,16 +129,22 @@ func main() {
 		parentName := r.URL.Query().Get("parentName")
 		childName := r.URL.Query().Get("childName")
 
+		tags := []ddtrace.StartSpanOption{}
```

Comment:
I should have asked this on a previous PR, sorry:  could you add a section in `/docs/weblog.md` explaining what this endpoint is supposed to do ?